Regression/WarpX-tests.ini

```
@@ -209,6 +209,22 @@ compileTest = 0
 doVis = 0
 analysisRoutine = Examples/Tests/btd_rz/analysis_BTD_laser_antenna.py
 
+[collider_diagnostics]
+buildDir = .
+inputFile = Examples/Tests/collider_relevant_diags/inputs_3d_multiple_particles
+runtime_params = warpx.abort_on_warning_threshold=high
+dim = 3
+addToCompileString =
+cmakeSetupOpts = -DWarpX_DIMS=3 -DCMAKE_BUILD_TYPE=Debug
```

`Debug` compilation mode was being used for local debugging purposes only, should not slip through:
```suggestion
cmakeSetupOpts = -DWarpX_DIMS=3
```